prov/kubernetes/executor.go

```diff
@@ -16,17 +16,22 @@ package kubernetes
 
 import (
 	"context"
+	"os"
 	"time"
 
-	"github.com/pkg/errors"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
+	"github.com/ystia/yorc/log"
```

Comment:
Ouch import ordering looks pretty damaged.
Usually we should have

std lib imports
// blank line
others dependencies imports
// blank line
github.com/ystia/yorc/... imports

Reply:
ah yes, will re-order these imports in next push